src/conv.rs

```diff
@@ -201,61 +192,127 @@ pub fn map_adapter_options<'a>(
 }
 
 pub fn map_device_descriptor<'a>(
-    _: &native::WGPUDeviceDescriptor,
+    des: &native::WGPUDeviceDescriptor,
     extras: Option<&native::WGPUDeviceExtras>,
 ) -> (wgt::DeviceDescriptor<Label<'a>>, Option<String>) {
+    let native_limits = unsafe { (*des.requiredLimits).limits };
```

Comment:
I guess we want to support a null-pointer here. Should we add this struct to `build.rs` to make it optional?

Reply:
Hmm, it doesn't appear to me that it should be optional.